src/dotnet/commands/dotnet-list/dotnet-list-package/LocalizableStrings.resx

```diff
@@ -0,0 +1,171 @@
+            </xsd:complexType>
+          </xsd:element>
+          <xsd:element name="data">
+            <xsd:complexType>
+              <xsd:sequence>
+                <xsd:element name="value" type="xsd:string" minOccurs="0" msdata:Ordinal="1" />
+                <xsd:element name="comment" type="xsd:string" minOccurs="0" msdata:Ordinal="2" />
+              </xsd:sequence>
+              <xsd:attribute name="name" type="xsd:string" use="required" msdata:Ordinal="1" />
+              <xsd:attribute name="type" type="xsd:string" msdata:Ordinal="3" />
+              <xsd:attribute name="mimetype" type="xsd:string" msdata:Ordinal="4" />
+              <xsd:attribute ref="xml:space" />
+            </xsd:complexType>
+          </xsd:element>
+          <xsd:element name="resheader">
+            <xsd:complexType>
+              <xsd:sequence>
+                <xsd:element name="value" type="xsd:string" minOccurs="0" msdata:Ordinal="1" />
+              </xsd:sequence>
+              <xsd:attribute name="name" type="xsd:string" use="required" />
+            </xsd:complexType>
+          </xsd:element>
+        </xsd:choice>
+      </xsd:complexType>
+    </xsd:element>
+  </xsd:schema>
+  <resheader name="resmimetype">
+    <value>text/microsoft-resx</value>
+  </resheader>
+  <resheader name="version">
+    <value>2.0</value>
+  </resheader>
+  <resheader name="reader">
+    <value>System.Resources.ResXResourceReader, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
+  </resheader>
+  <resheader name="writer">
+    <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
+  </resheader>
+  <data name="AppFullName" xml:space="preserve">
+    <value>List all package references of the project.</value>
+  </data>
+  <data name="CmdFramework" xml:space="preserve">
+    <value>FRAMEWORK | FRAMEWORK\RID</value>
+  </data>
+  <data name="CmdFrameworkDescription" xml:space="preserve">
+    <value>Chooses a framework to show its packages. Use the option multiple times for multiple frameworks.</value>
+  </data>
+  <data name="CmdOutdatedDescription" xml:space="preserve">
+    <value>Lists packages that have newer versions.</value>
+  </data>
+  <data name="CmdTransitiveDescription" xml:space="preserve">
+    <value>Lists transitive and top-level packages.</value>
+  </data>
+  <data name="MultipleProjectsFound" xml:space="preserve">
+    <value>The directory given includes multiple projects. Please specify one of the following projects:</value>
+  </data>
+  <data name="MultipleSolutionsFound" xml:space="preserve">
+    <value>The directory given includes multiple solutions. Please specify one of the following solutions:</value>
+  </data>
+  <data name="NoProjectsOrSolutions" xml:space="preserve">
+    <value>The given directory does not include any solution or project.</value>
```

Comment:
I don't think we have a common string for this error condition, since most of our commands rely on MSBuild to error if there's no solution or project file.

I think we should format this the same as the other two common strings: `A project or solution file could not be found in {0}. Specify a project or solution file to use.` (note: don't use the word *please* like the other two messages, we are removing them per Microsoft guidelines).